packages/suite/src/views/guide/Feedback.tsx

```diff
@@ -131,12 +131,47 @@ const ratingOptions: RatingItem[] = [
     },
 ];
 
+type FeedbackCategory = {
+    label: string;
+    value: Category;
+};
+
+const feedbackCategories: FeedbackCategory[] = [
+    { label: 'Dashboard', value: 'dashboard' },
+    { label: 'Accounts', value: 'account' },
+    { label: 'Settings', value: 'settings' },
+    { label: 'Send', value: 'send' },
+    { label: 'Receive', value: 'receive' },
+    { label: 'Trade', value: 'trade' },
+    { label: 'Other', value: 'other' },
+];
+
+// Router apps does not match 1:1 to Feedback Categories
+const getDefaultCategory = (category: string): FeedbackCategory => {
+    switch (category) {
+        case 'dashboard':
+        case 'start':
+            return feedbackCategories.filter(item => item.value === 'dashboard')[0];
+        case 'settings':
+        case 'version':
+        case 'bridge':
+            return feedbackCategories.filter(item => item.value === 'settings')[0];
+        case 'wallet':
+            return feedbackCategories.filter(item => item.value === 'send')[0];
+        case 'portfolio':
+            return feedbackCategories.filter(item => item.value === 'trade')[0];
+        default:
+            return feedbackCategories.filter(item => item.value === 'other')[0];
+    }
+};
+
```

Comment:
I'd much rather see something like
```
const feedbackCategories: FeedbackCategory[] = {
    dashboard: 'Dashboard',
    account: 'Accounts',
    ...
};

// Router apps does not match 1:1 to Feedback Categories
const getDefaultCategory = (app: typeof state.router.app): FeedbackCategory => {
    switch (category) {
                case 'dashboard':
                case 'start':
                                return feedbackCategories['dashboard'];
```

Reply:
You need `[{ label: 'Other', value: 'other' },...]` format for `react-select`. I changed it a little.